sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -657,14 +657,19 @@ def start_bundle(self):
       self.bq_io_metadata = create_bigquery_io_metadata(self._step_name)
     self.pending_jobs = []
 
-  def process(self, element, load_job_name_prefix, *schema_side_inputs):
+  def process(
+      self,
+      element,
+      load_job_name_prefix,
+      pane_info=beam.DoFn.PaneInfoParam,
+      *schema_side_inputs):
     # Each load job is assumed to have files respecting these constraints:
```

Comment:
is this back-compatible with adding pane_info?

Reply:
yes this should be fine. this is a standard parameter used in a DoFn process method. 

also, users don't actually use this PTransform directly